core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/ResourceProvider.scala

```diff
@@ -21,8 +21,6 @@ object ResourceProvider {
     new Cached(paths.cache, retries)
   }
 
-  val Default = new Cached(IdeProbePaths.Default.cache, 0)
```

Comment:
can you not delete these convenient `Default` vals that helps constructing other classes from code?

Reply:
Restored convenient Default vals